utils/importer.go

```diff
@@ -64,18 +65,29 @@ func MakeUniversalImporter(searchURLs []*url.URL) jsonnet.Importer {
 		BaseSearchURLs: searchURLs,
 		HTTPClient:     &http.Client{Transport: t},
 		cache:          map[string]jsonnet.Contents{},
+		alpha:          alpha,
 	}
 }
 
 type universalImporter struct {
 	BaseSearchURLs []*url.URL
 	HTTPClient     *http.Client
 	cache          map[string]jsonnet.Contents
+	alpha          bool // alpha features are enable only if true
 }
 
 func (importer *universalImporter) Import(importedFrom, importedPath string) (jsonnet.Contents, string, error) {
 	log.Debugf("Importing %q from %q", importedPath, importedFrom)
 
+	binary := false
+	if strings.HasPrefix(importedPath, "binary://") {
+		if !importer.alpha {
+			return jsonnet.Contents{}, "", fmt.Errorf(`"binary://" url prefix requires the --alpha flag`)
+		}
+		binary = true
+		importedPath = strings.TrimPrefix(importedPath, "binary://")
```

Comment:
.. Hrm.  I think this means a remote binary import looks like `import "binary://https://github.com/path"` ... ?

I think this probably needs to change to "https+binary://" or similar (at the scheme level) if we want this to graduate from alpha.

Reply:
Yeah i wanted this hack out so my sre team at work could exercise the helmChart feature asap

Reply:
I still have hopes the importbin statement gets released...

Reply:
> I think this probably needs to change to "https+binary://" or similar (at the scheme level) if we want this to graduate from alpha.

Just in case anyone acts on this comment, `https+binary` scheme is also a bad idea, since the scheme is 'sticky' for relative imports, and will be incorrect.  I think the "correct hack" is a URL fragment/anchor, which only impacts the way the client _interprets_ the content, and not the way the server or relative URLs are constructed.

ie: _If_ we choose to keep this hack, we should make it: `import "https://github.com/path#binary"`

Reply:
Interesting!

Yeah we definitely need to change this before graduating this from alpha.

I still hope importbin goes through before that